src/transformers/sinon.test.ts

```diff
@@ -293,6 +293,22 @@ describe('spies and stubs', () => {
 `
     )
   })
+
+  it('handles legacy .lastCall', () => {
+    expectTransformation(
+      `
+        import sinon from 'sinon'
+
+        apiStub.lastCall
+        apiStub.lastCall.args[1].data
+`,
+      `
+        apiStub.mock.calls[apiStub.mock.calls.length - 1]
```

Comment:
What is the use case here?

Why do you want `apiStub.mock.calls[apiStub.mock.calls.length - 1]` instead of `apiStub.mock.lastCall`?

Reply:
according to @catc, this is no longer needed (Jest 25 used to be in the mix but no longer is).  I'll revert this part shortly.